src/js/button.js

```diff
@@ -45,6 +45,7 @@ class Button extends ClickableComponent {
     attributes = assign({
       type: 'button', // Necessary since the default button type is "submit"
       'aria-live': 'polite' // let the screen reader user know that the text of the button may change
+      'tabindex' : 0 // add tabindex for keyboard accessibility to buttons
```

Comment:
For some reason, this code only adds tabindex for the three controls that the player complains are deprecated for using divs as buttons.  It doesn't add tabindex to actual button controls.